<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Setup</class>
 <widget class="QDialog" name="Dialog_Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FEMALE_lab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RIGHT_lab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hand_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="MALE_lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gender_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QSlider" name="gender_slider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hand_slider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QSlider" name="health_slider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LEFT_lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="forearmL_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elbow-to-Fingertip Length [in]</string>
   </property>
  </widget>
  <widget class="QLabel" name="STROKE_lab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_fill">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="upperarmL_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoulder-to-Elbow Length [in]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upperarmL_fill">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CTRL_lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="age_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID / Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_fill">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="health_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Health</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_fill">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="forearmL_fill">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LtoEE_fill">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LtoEE_lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elbow-to-EE Distance [in]</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="staircase_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Staircase Tests</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="match1D_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1-D Matching Tests</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="match2D_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2-D Matching Tests</string>
   </property>
  </widget>
  <widget class="QComboBox" name="match1Dtest1_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="match1Dtest1_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match1Dtest2_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match1Dtest3_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest1_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest2_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest3_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="match1Dtest2_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match1Dtest3_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match2Dtest3_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match2Dtest2_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match2Dtest1_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name_fill</tabstop>
  <tabstop>ID_fill</tabstop>
  <tabstop>age_fill</tabstop>
  <tabstop>health_slider</tabstop>
  <tabstop>gender_slider</tabstop>
  <tabstop>hand_slider</tabstop>
  <tabstop>upperarmL_fill</tabstop>
  <tabstop>LtoEE_fill</tabstop>
  <tabstop>forearmL_fill</tabstop>
  <tabstop>staircase_box</tabstop>
  <tabstop>shoulder_box_1</tabstop>
  <tabstop>elbow_box_1</tabstop>
  <tabstop>match1D_box</tabstop>
  <tabstop>match1Dtest1_combo</tabstop>
  <tabstop>vision_box_1</tabstop>
  <tabstop>shoulder_box_2</tabstop>
  <tabstop>elbow_box_2</tabstop>
  <tabstop>match1Dtest2_combo</tabstop>
  <tabstop>vision_box_2</tabstop>
  <tabstop>shoulder_box_3</tabstop>
  <tabstop>elbow_box_3</tabstop>
  <tabstop>match1Dtest3_combo</tabstop>
  <tabstop>vision_box_3</tabstop>
  <tabstop>shoulder_box_4</tabstop>
  <tabstop>elbow_box_4</tabstop>
  <tabstop>match2D_box</tabstop>
  <tabstop>match2Dtest1_combo</tabstop>
  <tabstop>vision_box_4</tabstop>
  <tabstop>match2Dtest2_combo</tabstop>
  <tabstop>vision_box_5</tabstop>
  <tabstop>match2Dtest3_combo</tabstop>
  <tabstop>vision_box_6</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Setup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Setup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
